libraries/AP_Proximity/AP_Proximity_Backend.h

```diff
@@ -22,6 +22,7 @@
 
 #define PROXIMITY_GND_DETECT_THRESHOLD 1.0f // set ground detection threshold to be 1 meters
 #define PROXIMITY_ALT_DETECT_TIMEOUT_MS 500 // alt readings should arrive within this much time
+#define PROXIMITY_BOUNDARY_3D_UPDATE_MS 1500 // we should check the 3D boundary faces after every this many ms
```

Comment:
maybe this is more of a timeout?  So perhaps rename it to PROXIMITY_BOUNDARY_3D_TIMEOUT_MS